main.cpp

```diff
@@ -55,10 +109,25 @@ struct Bakery
     int openingTime = 8;
     int closingTIme = 18;
 
-    bool checkOpen (int openingTime, int closingTime);
-    void bakeBread (int numOfOvens, int amountOfBread, int ovenNumber);
+    bool checkOpen (int opening, int closing, int current);
+    int bakeBread (int numOfOvens, int breadPerOven);
 };
 
+bool Bakery::checkOpen (int opening, int closing, int current)
+{
+    if (current > opening && current < closing)
+    {
+        return true;
+    }
+    return false;
+}
+
+int Bakery::bakeBread (int numOvens, int breadPerOven)
+{
+    return (amountOfBread-(numOvens*breadPerOven));
```

Comment:
add some space around your operators.  help the reader out
- [ ] add space around `-` and `*` to improve readability